### ISSUE TITLE
The Legacy REST API will move to a dedicated extension in WooCommerce 9.0

### ISSUE THREAD

**Issue by ALPBRCH** [woothemes/wc-api-php#343]
Hello,
Will there be an update for wc-api-php?

https://developer.woo.com/2023/10/03/the-legacy-rest-api-will-move-to-a-dedicated-extension-in-woocommerce-9-0/

Best regards
JI

**Comment by ALPBRCH** [comment]
completed

**Comment by joshuatf** [comment]
I don't believe there are any updates needed in this repository.  If you plan to continue using the legacy REST API, you will need to install that separately to continue using it.
